test/rubygems/test_gem_commands_exec_command.rb

```diff
@@ -215,6 +215,7 @@ def test_gem_with_platform_dependencies
 
   def test_gem_with_platform_and_platform_dependencies
     pend "extensions don't quite work on jruby" if Gem.java_platform?
+    pend "terminates on mswin" if Gem.win_platform? && ruby_repo?
```

Comment:
This time looks great 👍.

Reply:
I will rename `ruby_repo` to `ruby_core` because it's inconsistency of bundler's one.